core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/netezza/NetezzaDatabaseConnectionService.java

```diff
@@ -27,4 +27,9 @@ public String getDriver(NetezzaDatabaseConnection databaseConnection) {
     public Class<NetezzaDatabaseConnection> appliesTo() {
         return NetezzaDatabaseConnection.class;
     }
+
+    @Override
+    public String refactorLimitAndOffset(NetezzaDatabaseConnection databaseConnection, String query) {
+        return query;
+    }
```

Comment:
All XDatabaseConnectionService's extend either DatabaseConnectionService or SchemaDatabaseConnectionService (which extends DatabaseConnectionService). You could implement the default behavior in the DatabaseConnectionService class